src/test/java/com/revature/services/AddressServiceTest.java

```diff
@@ -88,6 +88,24 @@ void testUpdate() {
 		assertEquals(expected, actual);
 		verify(this.mockAddressRepo, times(1)).save(this.dummyAddress);
 	}
+	
+	@Test
+	void testUpdate_Failure_AddressNotFound() {
+		Set<User> users = this.dummyAddress.getUsers();
+		users.add(this.dummyUser);
+		AddressRequest updateRequest = new AddressRequest(this.dummyAddress.getStreet(),
+				this.dummyAddress.getSecondary(), this.dummyAddress.getCity(), this.dummyAddress.getZip(),
+				this.dummyAddress.getState());
+		this.dummyAddress.setUsers(users);
+		int id = 404;
+		given(this.mockAddressRepo.findById(id)).willReturn(Optional.empty());
+		try {
+			this.aServ.update(updateRequest, id, this.dummyUser);
+			fail("Expected an exception to be thrown");
```

Comment:
You should specify which exception type is expected to be thrown. In this case, it's `AddressNotFoundException`